src/main/java/com/zerobase/zbfintech/exception/ExceptionController.java

```diff
@@ -0,0 +1,28 @@
+package com.zerobase.zbfintech.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+@Slf4j
+public class ExceptionController {
+
+    @ExceptionHandler({CustomException.class})
+    public ResponseEntity<ExceptionResponse> customRequestException(final CustomException c){
```

Comment:
```suggestion
    public ResponseEntity<ExceptionResponse> customRequestException(final CustomException c) {
```
java convention 상 위와 같은 코드가 정상코드니 지킬 수 있도록 노력해보면 좋을 것 같아요